token/src/test/java/io/warp10/token/test/TestWriteToken.java

```diff
@@ -115,7 +115,7 @@ public void testTokenExpired() throws TException, InterruptedException {
 
     final QuasarTokenFilter tokenFilter = new QuasarTokenFilter(getConfig(), getKeyStore());
 
-    Thread.sleep(1);
+    Thread.sleep(2);
```

Comment:
Why is this in this PR?

Reply:
The test no longer passed. Now it's ok -> revert